index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Surf and Paddle | An Awesome Blog of Awesomeness</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="./main.css" />
+    <link href='http://fonts.googleapis.com/css?family=Muli:300' rel='stylesheet' type='text/css' />
```

Comment:
be consistent when you use `'` vs `"` in HTML. I recommend, for straight HTML, using the `"`